validation/validate/timeframe.go

```diff
@@ -0,0 +1,15 @@
+package validate
+
+import (
+	"time"
+
+	"github.com/target/goalert/validation"
+)
+
+func TimeFrame(fname string, since time.Time, until time.Time, timeFrame time.Duration) error {
```

Comment:
Add a comment for new public validation functions.

Also:
- Anything to be lifted from `schedule/validate.go`? We're calling this `TimeRange` there
- For this use, should it be validated based on an ISODuration instead?

Reply:
It looks like `validate/duration.go` can actually work for this instead of having a new function, and for this case using time.Duration might be easier since there isn't a good way to compare ISODurations right now.